src/app/lib/analyticsUtils/index.test.js

```diff
@@ -541,3 +542,42 @@ describe('getAtUserId', () => {
     expect(id).toBe(val);
   });
 });
+
+describe('getLibraryVersion', () => {
+  describe('For amp pages and canonical pages using client-side Javascript', () => {
+    const libraryVersionScenarios = [
+      {
+        platform: 'amp',
+        expected: 'simorgh',
+        summary: 'should return `simorgh` for amp',
+      },
+      {
+        platform: 'canonical',
+        expected: 'simorgh',
+        summary: 'should return `simorgh` for canonical',
+      },
+    ];
+
+    libraryVersionScenarios.forEach(({ platform, expected, summary }) => {
```

Comment:
I'm not sure I'm a fan of this. Can we keep these 2 tests explicit rather than being generated with a forEach. I think it's better not to add cognitive load when reading tests.

Reply:
@jroebu14  This has been fixed.

Reply:
Thanks :)